DDF.xml

```diff
@@ -2896,12 +2896,14 @@ These LwM2M Object Resources MUST only be changed by a LwM2M Bootstrap-Server or
     <Item>
         <ObjectID>10300</ObjectID>
         <URN>urn:oma:lwm2m:x:10300</URN>
-        <Name>Reserved</Name>
-        <Description>TBC</Description>
-        <Owner>TBC</Owner>
+        <Name>LWM2M Meta Object</Name>
+        <Description>Use this meta object to provide one or more locations hosting XML data
+ 	  model definitions for reusable or private and vendor-specific devices,
+ 	  such as those in the range 26241-42768.</Description>
+        <Owner>TU</Owner>

```

Comment:
Please amend the name of the Owner as TAU  (which is the official abbreviation of Tampere University), Thank you